### PR TITLE
Updated changes around Module stability related to iOS and Swift.

### PR DESCRIPTION
**Pull request by Nautiyalsachin**
* I have removed Swift version as now it won't be targeting any specific version like Swift 5.0.
* Updated podspec version to the same as flutter plugin version - 1.1.3
* Updated Readme to the latest version. 

You can also suggest me the missing changes if there is any.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -21,7 +21,7 @@ This plugin is available on Pub: [https://pub.dev/packages/razorpay_flutter](htt
 Add this to `dependencies` in your app's `pubspec.yml`
 
 ```yaml
-razorpay_flutter: ^1.1.1
+razorpay_flutter: ^1.1.3
```
Comment (ChintanAcharya) [inline README.md:24]: Why not `1.1.2`?

Comment (Nautiyalsachin) [reply README.md:24]: It was bumped earlier to 1.1.2 and the readme was not updated.

Comment (ChintanAcharya) [reply README.md:24]: My bad, I thought it was pubspec.yml

File: ios/razorpay_flutter.podspec [hunk 2]
```diff
@@ -3,7 +3,7 @@
 #
 Pod::Spec.new do |s|
   s.name             = 'razorpay_flutter'
-  s.version          = '1.1.1'
+  s.version          = '1.1.3'
```
Comment (ChintanAcharya) [inline ios/razorpay_flutter.podspec:6]: Update the gradle version as well.
https://github.com/razorpay/razorpay-flutter/blob/master/android/build.gradle#L2

**Pull request merged**